client/src/components/admin/Home.vue

```diff
@@ -118,7 +118,7 @@
 <script>
 export default {
     props: {
-        isToolshedInstalled: {
+        isToolShedInstalled: {
```

Comment:
Thank you for these fixes. I think this name should be fixed at: https://github.com/galaxyproject/galaxy/pull/14505/files#diff-e8e4ee15b798479e71d049302fc72771d8c16416969dc24737e6e08858eabb3bR93